external/kafka-assembly/pom.xml

```diff
@@ -66,13 +66,16 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>com.sun.jersey</groupId>
-      <artifactId>jersey-server</artifactId>
-      <scope>provided</scope>
+      <groupId>org.glassfish.jersey.core</groupId>
+      <artifactId>jersey-client</artifactId>
```

Comment:
Just poked a bit through the Git history and it's to not include the jars that the assembly already provides. Although we no longer use an assembly jar... this still applies since we'll get org.glassfish.jersey from the main Spark classpath as we were before, so I'll put provided back in.